tidb-cloud: clarify backup behavior during data import and scaling (#23244)
pingcap/docs: tidb-cloud/backup-and-restore.md

        > **Note**
        >

>         > - During data import, automatic backup jobs are delayed. In addition, you cannot run manual backups during data import.

Reviewer: ```suggestion
        > - During data import, TiDB Cloud delays automatic backup jobs, and you cannot run manual backups.
```

>         > - During cluster scaling, During cluster scaling, automatic backup jobs are delayed. If the delay exceeds 30 minutes, TiDB Cloud skips the backup. **DO NOT** run manual backups during cluster scaling.

Reviewer: ```suggestion
        > - During cluster scaling, automatic backup jobs are delayed. If the delay exceeds 30 minutes, TiDB Cloud skips the backup. **DO NOT** run manual backups during cluster scaling.
```

>         > - During cluster scaling, automatic backup jobs are delayed. If the delay exceeds 30 minutes, TiDB Cloud skips the backup. To help ensure scheduled backups run successfully, avoid frequent cluster scaling around the scheduled backup time. **DO NOT** run manual backups during cluster scaling.

Reviewer: ```suggestion
        > - During cluster scaling, TiDB Cloud delays automatic backup jobs. If the delay exceeds 30 minutes, TiDB Cloud skips the scheduled backup. To help ensure scheduled backups run successfully, avoid frequent cluster scaling around the scheduled backup time. **DO NOT** run manual backups during cluster scaling.
```
